### ISSUE TITLE
mkv:libx265 movie_codec does not save files as mvk

### ISSUE THREAD

**Issue by jkemp814** [Motion-Project/motion#945]
Can the mkv:libx265 movie_codec be made to save files as mkv?
I noticed when using mvk movie codec and H.264 encoder it does save as mkv
mkv:libx265 is using H.265 codec but saves as .mp4
Which is a problem because mp4 files when corrupted cannot be recovered.
.mkv can be recovered

**Comment by genius3000** [comment]
I'm using `movie_codec mkv:libx265` and files are saved as .mkv. Do your logs show anything that might be a clue as to why yours is using .mp4?

**Comment by jkemp814** [comment]
Hmmm, let me check

**Comment by jkemp814** [comment]
I think I just found something. I'm using the movie_extpipe option to pass the load off on another thread and the ffmpeg string had .mp4 as the file name

movie_extpipe ffmpeg -y -f rawvideo -pix_fmt yuv420p -video_size %wx%h -framerate %fps -i pipe:0 -vcodec libx265 -preset ultrafast -f mp4 %f.mp4

So I changed to:
movie_extpipe ffmpeg -y -f rawvideo -pix_fmt yuv420p -video_size %wx%h -framerate %fps -i pipe:0 -vcodec libx265 -preset ultrafast -f mp4 %f.mkv

Thanks for the response. It verified that it should be doing what it suppose to do.

Seems to working as intended.
